pkg/synthesis/tests_expected_output/ExampleDumbeldore.txt

```diff
@@ -0,0 +1,12 @@
+category: Application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+symbolic inbound rules:
+	0. action: allow paths: (group  = DumbledoreAll) to (*)
```

Comment:
is rule scope / category scope going to be reflected in symbolic version?
or is it going to be combined as src/dst symbolic expression?

Reply:
The latter. 
Pointer to the original rule is kept, but that's it.

Regarding the scope - basically its just another term that can be differed to the last stages before the actual synthesis.